packages/databricks-vscode/src/bundle/GenerateBundle.ts

```diff
@@ -21,16 +69,9 @@ export async function generateBundleSchema(cli: CliWrapper) {
         redHatYamlSchemaApi.registerContributor(
             "dabs",
             (resource: string) => {
-                const validFileNames: string[] = [
-                    "databricks.yml",
-                    "databricks.yaml",
-                    "bundle.yml",
-                    "bundle.yaml",
-                ];
-                for (const name of validFileNames) {
-                    if (path.basename(resource) === name) {
-                        return rootConfigSchemaUri;
-                    }
+                const resourceUri = Uri.parse(resource);
+                if (configFiles.has(resourceUri.path)) {
+                    return rootConfigSchemaUri;
```

Comment:
Alternatively, why can't we just call the updateFileGlobs here? It is not a very expensive function (for reasonable bundles) and less compute intensive than watchers.

Reply:
This code is executed whenever a file is opened. I'd like to keep that code path as fast as possible.

Reply:
This need not be too expensive. All we need to do is match the `resourceUri` with all the include paths in the root bundle.yaml. 

That is mostly string matching and regex matching for globs. We shouldn't even need filesystem access if we have a single watcher on root bundle.yaml keeping the list of includes updated.